src/DI/ContainerBuilder.php

```diff
@@ -174,24 +168,21 @@ public function getAliases()
 
 
 	/**
+	 * @param  string[]
 	 * @return self
 	 */
-	public function setClassName($name)
```

Comment:
I never got to migrate and it's not really needed for me anymore (since you've solved it natively), but I was using custom container naming back in the day in tests, so it wouldn't collide.